### ISSUE TITLE
How to run tests with sinon?

### ISSUE THREAD

**Issue by marr** [webpack/karma-webpack#43]
Hi, 
I am trying to run a test package and noticing that webpack still seems to bundle things in node_modules even though I am using the following karma/webpack config: https://gist.github.com/fb66622fa8284c76e6d9

An example test file:
```
'use strict';

var React = require('react/addons');
//var sinon = require('sinon');
```
React gets bundled, without issue even though I'd think that excludes would ignore it.
The issue is when I uncomment the sinon test, I get several critical issues:

```
WARNING in ./~/sinon/lib/sinon.js
Critical dependencies:
39:25-32 require function is used in a way, in which dependencies cannot be statically extracted
 @ ./~/sinon/lib/sinon.js 39:25-32

WARNING in ./~/sinon/lib/sinon/behavior.js
Critical dependencies:
353:25-32 require function is used in a way, in which dependencies cannot be statically extracted
 @ ./~/sinon/lib/sinon/behavior.js 353:25-32
```

Am I just doing things wrong by expecting mocha tests to run like they would on the server?

**Comment by ColCh** [comment]
Just use regular __sinon__ plugin for karma - sinon's API will be global within tests

**Comment by marr** [comment]
Thanks @ColCh I ended up doing that and its working great. Here's an example if anyone wants to see it. https://github.com/bitwise/status

**Comment by josmardias** [comment]
@marr your link is broken

**Comment by marr** [comment]
@josmardias I will look at pushing it back up somewhere. Meantime, you should be able to use https://github.com/kmees/karma-sinon-chai

**Comment by ColCh** [comment]
@josmardias use [karma-sinon](https://www.npmjs.com/package/karma-sinon) or [karma-sinon-chai](https://www.npmjs.com/package/karma-sinon-chai)

If you need your unique setup, you can simple require *sinon* within *webpack*, and use `modules: noparse: [ /sinon\.js/ ]` within your *webpack* config (that worked for me). Documentation for [module.noParse](https://webpack.github.io/docs/configuration.html#module-noparse)

**Comment by josmardias** [comment]
I'll try it, thanks!

**Comment by tschaub** [comment]
Came across this same issue and found that it is fixed in `sinon@2.0.0-pre.5`.
